gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/tests/integration/KotlinCompatibilityTest.kt

```diff
@@ -23,6 +23,22 @@ class KotlinCompatibilityTest : GradlePluginTestBase() {
     @Test
     fun testKotlinJsMpp_1_7_20() = testJsMpp("1.7.20")
 
+    @Suppress("RedundantUnitExpression")
+    @Test
+    fun testKotlinJs_shows_warning_for_androidx_compose_compiler() = testProject(
+        TestProjects.customCompilerArgs, defaultTestEnvironment.copy(
+            kotlinVersion = "1.8.22",
+            composeCompilerPlugin = "\"androidx.compose.compiler:compiler:1.4.8\"",
+            composeCompilerArgs = "\"suppressKotlinVersionCompatibilityCheck=1.8.22\""
+        )
+    ).let {
+        it.gradle(":compileKotlinJs").checks {
+            check.taskSuccessful(":compileKotlinJs")
+            check.logContains("WARNING: You are using the 'androidx.compose.compiler' compiler plugin in your Kotlin multiplatform project.")
```

Comment:
Maybe using a const from the main source set could be more convenient in case anyone wants to edit the message.

Reply:
The const is not possible because we use a template message. But I made a function internal, so we call it in the tests as well.